docs/bundles/plain.md

```diff
@@ -2,21 +2,52 @@
 
 ## Overview
 
-This document is meant to define the `plain` bundle format as a reference for those publishing `plain` bundles
-for use with RukPak. For more information on the concept of a bundle, click [here](https://github.com/operator-framework/rukpak#bundle).
+This document is meant to define the plain bundle format as a reference for those publishing plain bundles for use with
+RukPak. A bundle is a collection of Kubernetes resources that are packaged together for the purposes of installing onto
+a Kubernetes cluster. A bundle can be unpacked onto a running cluster, where controllers can then create the underlying
+content embedded in the bundle. The bundle can be used as the underlying `spec.source` for
+a [Bundle](https://github.com/operator-framework/rukpak#bundle) resource.
 
-A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory.
+A plain bundle is simply a collection of static, arbitrary, Kubernetes YAML manifests in a given directory. A plain
+bundle can consist of a container image, a directory in a git repository, or any other content source that
+the [plain bundle provisioner](../provisioners/plain.md)
```

Comment:
This reference is a bit of a tautology and does not help new users learn what packaging types are supported. Perhaps enumerate all the supported types and link out to this for more info?

Reply:
There should be a section a bit below this one that mentions the supported packaging types. I'll move it up to be a bit closer to this section.